manifest-rhel-9.0.yaml

```diff
@@ -114,7 +114,11 @@ packages:
  # We include the generic release package and tweak the os-release info in a
  # post-proces script
  - redhat-release
-
+ # We have two versions of multipath in the rhel9 repo,
+ # The first version 8.7-7.el9 is causing kola to fail
+ # Let's lock it to 8.7-7.el9_0.1 until a new version is 
+ # available
```

Comment:
Is there an issue/bug we can paste here to track reverting this pin?

Aside: although RHCOS doesn't have base lockfiles, it should still be possible to have a `manifest-lock.overrides.yaml` file we put this in and have cosa pick it up. I think before we can do that though we'd need to make lockfiles variant-aware somehow, e.g. including it in the lockfile names.

Reply:
There is an internal issue.  We probably should open one upstream, and other to enable the `manifest-lock.overrides.yaml`

Reply:
https://github.com/openshift/os/issues/1110